src/select/parts/trigger.tsx

```diff
@@ -47,11 +46,7 @@ const Trigger = React.forwardRef(
     let triggerContent = null;
     if (!selectedOption) {
       triggerContent = (
-        <span
-          aria-disabled="true"
-          className={clsx(styles.placeholder, styles.trigger)}
-          id={triggerProps.ariaLabelledby}
```

Comment:
This id now(without this pr) only set when there is selectedOption https://github.com/cloudscape-design/components/blob/main/src/select/utils/use-select.ts#L167 . With this pr, it behaves as button content, will be read out when no aria-label/aria-labelledby set.